History.md

```diff
@@ -199,6 +199,10 @@
   and `html` files have been updated to use this new API. Read more on the
   [Wiki page](https://github.com/meteor/meteor/wiki/Build-Plugins-API).
 
+* There is a new API for adding assets to packages, `api.addAssets`. It works
```

Comment:
A few lines up is a reference to saying you should say `isAsset: true`. Update that.

(Also, more generally, I'm concerned that the backwards-incompatible change is buried halfway through history. I know there's an upgrade guide; will it be linked from History?)

Also maybe this should be more clear that the old `{isAsset}` flag is deprecated?